garage.com/src/main/java/xpathproject.txt

```diff
@@ -0,0 +1,33 @@
+1. Page: /text-box. Write the xpath that will return the ‘Full Name’ textbox using the ID attribute. 
+Hint: This element has the ‘input’ tag.
+	"//input[@id='userName']"
+
+
+2. Page: /buttons. Write the xpath that will return all elements on the page with the ‘button’ tag.
```

Comment:
//button

Reply:
The CSS Selector part is in RkQuintrixTraining repository. The pull request had been submitted already and you already approved it. Thank you.